internal/schedule_client.go

```diff
@@ -476,6 +495,10 @@ type (
 	ScheduleUpdate struct {
 		// Schedule - New schedule to replace the existing schedule with
 		Schedule *Schedule
+
+		// TypedSearchAttributes - Optional indexed info that can be used for querying via the List schedules APIs.
+		// The key and value type must be registered on Temporal server side.
+		TypedSearchAttributes *SearchAttributes
```

Comment:
Should clarify what `nil` vs empty means here IMO (the stuff on description should probably be here)